platform/lib/routers/go.test.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const request = require('supertest');
+
+const app = express();
+const router = require('./go.js');
+app.use(router);
+
+test('redirects simple golink', (done) => {
+  request(app)
+      .get('/tools')
```

Comment:
Super that unit tests are included! :-D
But it would be better not to rely on the content in the regular config file, since this should be able to change without adapting the unit tests.
We already have an example where we mocked the yaml here: platform/lib/middleware/redirects.test.js

Reply:
Done - thanks, I didn't know it was that easy to inject a mock config.